README.md

```diff
@@ -1 +1,55 @@
-# perl-Mojo-WebSocketProxy
\ No newline at end of file
+# perl-Mojo-WebSocketProxy
+[![Build Status](https://travis-ci.org/binary-com/perl-Mojo-WebSocketProxy.svg?branch=master)](https://travis-ci.org/binary-com/perl-Mojo-WebSocketProxy) 
+
+Using this module you can forward websocket JSON-RPC 2.0 requests to RPC server.
```

Comment:
@evostrov use

https://metacpan.org/pod/distribution/Pod-Markdown/bin/pod2markdown

to generate readme from pod